Formula/plank.rb

```diff
@@ -0,0 +1,19 @@
+class Plank < Formula
+  plank_version = "v1.2"
+  desc "Framework for generating immutable model objects"
+  homepage "https://pinterest.github.io/plank/"
+  url "https://github.com/pinterest/plank.git", :tag => plank_version
+  head "https://github.com/pinterest/plank.git"
+
+  depends_on :xcode => ["9.0", :build]
+  depends_on "kylef/formulae/swiftenv" => [:build]
```

Comment:
This is not permitted in `homebrew/core`. Will either have to be added as a new formula or vendored in as a resource, but I'm not sure what you're aiming for here with `swiftenv`? Kinda unusual to use a version manager as a build tool.